### ISSUE TITLE
restrict visibility of URL link for 10 days on newly created account profiles

### ISSUE THREAD

**Issue by rahaeli** [dreamwidth/dw-free#2795]
The vast majority of our spam is SEO spammers creating an account, wodging their spammy URL into the URL field of the profile, posting a spammy entry or two, and never touching the account again. We are generally very good at finding them within a reasonable amount of time, but a simple change that might make us less appealing for spammer account creation: people should be able to add the URL to their account profile whenever, but it shouldn't display to anyone other than the owner of the account for the first 10 days after account creation. (Ideally, with a note next to the field so if the user looks at their profile and doesn't see the URL, they don't think it's a bug: "Other people will not see this link on your profile until your account is at least 10 days old.")

**Comment by kareila** [comment]
I was going to say I don't know that this would make our spamwhacking efforts any more effective, but if it would discourage spammers from using our site, I'm all for it. That will be pretty easy for me to add once my converted profile page PR gets merged. (I've finished it but I was waiting for the post-push fixes to die down before submitting it.) I would also want to continue displaying the link to logged-in users with spam suspend privileges, for spam identification purposes.

Would we also want the link not to display in the profile module of the user's journal? That might be a bit trickier to implement.

**Comment by rahaeli** [comment]
Yeah, if it's possible. If not easy to just block the website there I'd also be fine with removing the website link from the profile module entirely or only displaying the whole profile module to the logged-in user for the first 10 days.
